### PR TITLE
chore: update mayastor pool CR definition file

### PR DESCRIPTION
**Pull request by rohan2794**
This PR update the mayastor pool CR definition file from control plane repo   https://github.com/mayadata-io/mayastor-control-plane/blob/develop/chart/templates/mayastorpoolcrd.yaml ) and fixes primitive msp state test due to CR changes.

JIRA: https://mayadata.atlassian.net/browse/MQ-2680

### CODE REVIEW

**Review by chriswldenyer — APPROVED** [review]
LGTM, but could common.MayastorPool also be converted to use uint64 to avoid the casts?

**Review by chriswldenyer — APPROVED** [review]
LGTM

File: src/primitive_msp_state/util.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func verifyMspState(poolName string, crPool v1alpha1.MayastorPool,
 func verifyMspCapacity(poolName string, crPool v1alpha1.MayastorPool,
 	grpcPool mayastorclient.MayastorPool) bool {
 	var status bool
-	if crPool.Status.Capacity == int64(grpcPool.Capacity) {
+	if int64(crPool.Status.Capacity) == int64(grpcPool.Capacity) {
```
Comment (blaisedias) [inline src/primitive_msp_state/util.go:101]: casting both values it int64 feels wrong, cast to uint64 (if required)

File: src/primitive_msp_state/util.go [hunk 2]
```diff
@@ -114,7 +114,7 @@ func verifyMspCapacity(poolName string, crPool v1alpha1.MayastorPool,
 func verifyMspUsedSpace(poolName string, crPool v1alpha1.MayastorPool,
 	grpcPool mayastorclient.MayastorPool) bool {
 	var status bool
-	if crPool.Status.Used == int64(grpcPool.Used) {
+	if int64(crPool.Status.Used) == int64(grpcPool.Used) {
```
Comment (blaisedias) [inline src/primitive_msp_state/util.go:117]: Same here

File: src/primitive_msp_state/util.go (hunk 1 above)
Comment (rohan2794) [reply src/primitive_msp_state/util.go]: Done

**Review by blaisedias — APPROVED** [review]

**Comment by rohan2794** [comment]
bors merge

**Comment by rohan2794** [comment]
bors merge

**Comment by rohan2794** [comment]
bors merge

**Pull request merged**